cmd/docker/docker.go

```diff
@@ -325,6 +326,25 @@ func tryPluginRun(ctx context.Context, dockerCli command.Cli, cmd *cobra.Command
 	return nil
 }
 
+// registerForceExitGoroutine registers a goroutine that will force exit the
+// process after 3 SIGTERM/SIGINT signals.
+func registerForceExitGoroutine(ctx context.Context, w io.Writer) {
+	// setup a signal handler to force exit after 3 SIGTERM/SIGINT
+	go func() {
+		<-ctx.Done()
```

Comment:
nit: i think we could leave a small comment here as to why we're waiting on this `ctx.Done()` chan, and why we check for `count >= 2` below